### DISCUSSION TITLE
Can't get normal size window for MountWizzard

### DISCUSSION THREAD

**Discussion by Tom-H3** [mworion/MountWizzard4#375] [Feature Request]
I was able to get a beta version of MountWizzard working, but in window that was too small for me to use. I ran a suggested script, but I could not make it work.Here is the result:
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> python startup.pyz--scale 1.4
C:\Users\tom40\AppData\Local\Programs\Python\Python310\python.exe: can't open file 'C:\\Users\\tom40\\Desktop\\New folder\\venv\\Lib\\site-packages\\mw4\\startup.pyz--scale': [Errno 2] No such file or directory
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> startup.pyz scale 1.4
startup.pyz : The term 'startup.pyz' is not recognized as the name of a cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ startup.pyz scale 1.4
+ ~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (startup.pyz:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> python startup.pyz --help
C:\Users\tom40\AppData\Local\Programs\Python\Python310\python.exe: can't open file 'C:\\Users\\tom40\\Desktop\\New folder\\venv\\Lib\\site-packages\\mw4\\startup.pyz': [Errno 2] No such file or directory
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4>

**Comment by mworion** [comment]
Hi Tom,

the command is

python startup.pyz —scale 1.4 —dpi 96

As I looked to your mail I saw that there might be no space between 
“startup.pyz” and “—scale 1.4”
There shoud be no space between “—“ and “scale”, but another space between 
“—scale” and 1.4.

Hope this helps.

Merry christmas !
Michel

Gesendet mit der Telekom Mail App
<http://www.t-online.de/service/redir/emailmobilapp_ios_smartphone_footerlink.htm>

-----Original-Nachricht-----
Von: Tom-H3 ***@***.*** ***@***.***> >
Betreff: [mworion/MountWizzard4] Can't get normal size window for 
MountWizzard (Discussion #375)
Datum: 23.12.2024, 05:44 Uhr
An: mworion/MountWizzard4 ***@***.***
***@***.***> >
CC: Subscribed ***@***.***
***@***.***> >

I was able to get a beta version of MountWizzard working, but in window 
that was too small for me to use. I ran a suggested script, but I could not 
make it work.Here is the result:
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> python 
startup.pyz--scale 1.4
C:\Users\tom40\AppData\Local\Programs\Python\Python310\python.exe: can't 
open file 'C:\Users\tom40\Desktop\New 
folder\venv\Lib\site-packages\mw4\startup.pyz--scale': [Errno 2] No such 
file or directory
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> 
startup.pyz scale 1.4
startup.pyz : The term 'startup.pyz' is not recognized as the name of a 
cmdlet, function, script file, or operable
program. Check the spelling of the name, or if a path was included, verify 
that the path is correct and try again.
At line:1 char:1

    * startup.pyz scale 1.4
    *   + CategoryInfo          : ObjectNotFound: (startup.pyz:String) [], CommandNotFoundException
        + FullyQualifiedErrorId : CommandNotFoundException


PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4> python 
startup.pyz --help
C:\Users\tom40\AppData\Local\Programs\Python\Python310\python.exe: can't 
open file 'C:\Users\tom40\Desktop\New 
folder\venv\Lib\site-packages\mw4\startup.pyz': [Errno 2] No such file or 
directory
PS C:\Users\tom40\Desktop\New folder\venv\Lib\site-packages\mw4>

—
Reply to this email directly, view it on GitHub
<https://github.com/mworion/MountWizzard4/discussions/375> , or 
unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABSTMFA2ZV2OXESBXWQQDRD2G6IJPAVCNFSM6AAAAABUCMFNDCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXG4ZTGNZSGE> 
.
You are receiving this because you are subscribed to this thread.Message 
ID: ***@***.***>

**Comment by Tom-H3** [comment]
Thanks Michael,
Are  those double dashes “—“?
Merry Christmas to you too.
Tom

**Comment by Tom-H3** [comment]
Hello again Michael,
I get “error2 no such file or directory”.
Could it be because I have an AMD based PC?
Regards,
Tom

**Comment by mworion** [comment]
Hi Tom,
yes they are double dashes.
Michel

**Comment by BRIELL66** [comment]
Hi,  
when I copy that command
python startup.pyz —scale 1.4 —dpi 96
then paste it onto the Win11 command line it does not work, no error message, MW4 does not start.
   
When I copy-paste just   python startup.pyz   then it does start up ok but far too small to read.
On the command line — looks like - but when I copy-paste it from the back into this message, it again looks like —
Brian

**Comment by mworion** [comment]
Brian,

please type the commands as copy might have some automatic replacements for readability.
So it is

python startup.pyz 

and then 

space, minus, minus, "scale", space, "1", dot, "4", enter. 

Michel

![Bildschirmfoto 2025-01-06 um 13 32 17](https://github.com/user-attachments/assets/3c0f10e1-e4e6-4554-8ccb-18a72803e8ea)
![Bildschirmfoto 2025-01-06 um 13 32 56](https://github.com/user-attachments/assets/d4fc34d9-7975-4dc4-96bf-aec9b4bb61d4)

**Reply by BRIELL66** [reply]
Michel, 
that worked, thank you very much for your help!
